geomesa-geojson/geomesa-geojson-api/src/main/scala/org/locationtech/geomesa/geojson/query/GeoJsonQuery.scala

```diff
@@ -219,156 +252,168 @@ object GeoJsonQuery {
+    override def toFilter(idPath: Option[Seq[PathElement]], dtgPath: Option[Seq[PathElement]]): Filter =
+      ff.dwithin(ff.property(prop), ff.literal(geometry), dist, units)
+
+    override def toString =
+      s"""{"$prop":{"$$dwithin":${printJson(geometry)}, "$$dist":$dist, "$$unit":"$units"}}"""
   }
 
+  object Dwithin {
+    def apply(geometry: Geometry, distance: Double, units: String): Dwithin =
+      Dwithin(defaultGeom, geometry, distance, units)
+  }
+
+
   /**
     * Spatial bounding box
     *
-    * @param path property to evaluate
+    * @param prop property to evaluate
     * @param xmin min x value
     * @param ymin min y value
     * @param xmax max x value
     * @param ymax max y value
     */
-  case class Bbox(path: Seq[PathElement], xmin: Double, ymin: Double, xmax: Double, ymax: Double) extends GeoJsonQuery {
+  case class Bbox(prop: String, xmin: Double, ymin: Double, xmax: Double, ymax: Double) extends GeoJsonQuery {
     override def toFilter(idPath: Option[Seq[PathElement]], dtgPath: Option[Seq[PathElement]]): Filter =
-      ff.bbox(filterAttribute(path, None), xmin, ymin, xmax, ymax, "4326")
-    override def toString = s"""{"${JsonPathParser.print(path, dollar = false)}":{"$$bbox":[$xmin,$ymin,$xmax,$ymax]}}"""
+      ff.bbox(ff.property(prop), xmin, ymin, xmax, ymax, "4326")
+
+    override def toString = s"""{"$prop":{"$$bbox":[$xmin,$ymin,$xmax,$ymax]}}"""
   }
 
   object Bbox {
     def apply(xmin: Double, ymin: Double, xmax: Double, ymax: Double): Bbox =
-      Bbox(GeoJsonGeometryPath, xmin, ymin, xmax, ymax)
-    def apply(path: String, xmin: Double, ymin: Double, xmax: Double, ymax: Double): Bbox =
-      Bbox(JsonPathParser.parse(path), xmin, ymin, xmax, ymax)
+      Bbox(defaultGeom, xmin, ymin, xmax, ymax)
   }
 
   /**
     * Equality
     *
-    * @param path property to evaluate
+    * @param prop  property to evaluate
     * @param value value to compare with property value
     */
-  case class Equals(path: Seq[PathElement], value: Any) extends GeoJsonQuery {
+  case class Equals(prop: String, value: Any) extends GeoJsonQuery {
     override def toFilter(idPath: Option[Seq[PathElement]], dtgPath: Option[Seq[PathElement]]): Filter = {
-      if (idPath.exists(_ == path)) {
-        val fids = value match {
-          case v: Iterable[_] => v.map(_.toString).toSeq
-          case v => Seq(v.toString)
-        }
-        ff.id(fids.map(ff.featureId): _*)
-      } else {
-        ff.equals(filterAttribute(path, dtgPath), ff.literal(value))
-      }
+      //      if (idPath.exists(_ == path)) {
+      //        val fids = value match {
+      //          case v: Iterable[_] => v.map(_.toString).toSeq
+      //          case v => Seq(v.toString)
+      //        }
+      //        ff.id(fids.map(ff.featureId): _*)
+      //      } else {
+      //        ff.equals(filterAttribute(path, dtgPath), ff.literal(value))
+      //      }
```

Comment:
I think the code should go, and I should have deleted it rather than pushing commented out code. I do wonder about the original intent though, and ask @elahrvivaz if it's something we should keep.